src/posts/posts.controller.ts

```diff
@@ -30,21 +27,8 @@ export class PostsController {
   }
 
   @Patch(':id')
-  public async addComment(
-    // @Param('id') postId: string,
-    @Body() body: InsertCommentDto,
-  ): Promise<string> {
-    // await this.postsService.updatePost(postId, body);
-
-    console.log('body: ', body);
-
-    console.log('post controller postId: ', body.postId);
-
-    // const updatedPost = await this.postsService.getSinglePost(postId);
+  public async addComment(@Body() body: InsertCommentDto): Promise<string> {
     const generatedId = await this.commentsService.insertComment(body);
-    // const updatedPost = await this.postsService.updatePost(postId, body);
-    // return { updatedPost, generatedId };
-    // return updatedPost;
 
     return generatedId;
   }
```

Comment:
Same issues here, function return types. if it looks like:
```
    const posts = await this.postsService.getPosts();
    return posts;
```

Change it to 
`
    return await this.postsService.getPosts();`